runtime/src/main/cpp/ObjCExport.h

```diff
@@ -3,6 +3,8 @@
 
 #if KONAN_OBJC_INTEROP
 
+#if __OBJC__
```

Comment:
Why is this required?

Reply:
To be includable into `Memory.cpp`.

Reply:
Well, this gives one more reason for https://github.com/JetBrains/kotlin-native/pull/4446#discussion_r511776827.

Reply:
Done